### DISCUSSION TITLE
Install sponge utility to github actions automatically

### DISCUSSION THREAD

**Discussion by onnimonni** [community/community#56831] [Actions]
### Select Topic Area

Product Feedback

### Body

Hey!

For example `jq` can't replace files in place and the recommended way to do this is with `sponge` which can be installed with `sudo apt-get install moreutils`.

Similiarly as having `curl` and `jq` available as default could we also include `sponge`?

See more: https://stackoverflow.com/a/36577521

**Comment by favna** [comment]
Giving this thread a bump because I ran into it just now as well. `> tmp && mv tmp src/tsconfig.json` is not as nice as `| sponge tsconfig.base.json`.

**Comment by nagasriramnani** [comment]
Response Comment:

Hello,



Your point about jq not being able to replace files in-place is well-taken. The workaround involving sponge is indeed a common solution among the community, as it allows users to read the entire input before writing to the output file.
 who need to use sponge, you've rightly pointed out that it can be installed using:

bash
Copy code
sudo apt-get install moreutils
We will update this discussion with any new information about potential changes to the default environment as it becomes available.

Thank you once again for your contribution!

**Reply by favna** [reply]
This sounds extremely AI generated... are you even a human or are you a bot.
